src/clarity-angular/layout/tabs/tabs-service.ts

```diff
@@ -19,10 +23,46 @@ export class TabsService {
         return this._children;
     }
 
+    get activeTab() {
+        return this.children.find((tab: Tab) => {
+            return tab.active;
+        });
+    }
+
+    get overflowTabs() {
+        return this.children.filter((tab: Tab) => {
+            return tab.tabLink.inOverflow === true;
+        });
+    }
+
+    registerLink(link: any) {
+        this._links.push(link);
+    }
+
+
+    unregisterLink(link: any) {
+        const index = this._links.indexOf(link);
+        if (index > -1) {
+            this._links.splice(index, 1);
+        }
+    }
+
     unregister(tab: Tab) {
         const index = this.children.indexOf(tab);
         if (index > -1) {
             this.children.splice(index, 1);
         }
+
+    }
+
+    renderView() {
+        this._links.forEach((link: TabLinkDirective) => {
+           if (link.inOverflow) {
+               this.overflowTabLinksContainerRef.createEmbeddedView(link.templateRefContainer.template);
+           } else {
+               this.inlineTabLinksContainerRef.createEmbeddedView(link.templateRefContainer.template);
```

Comment:
Hmm actually I think it's much cleaner and simpler to move this logic into `Tabs`. Take a look and lmk what you think!